Source/ACE/Entity/Creature.cs

```diff
@@ -115,6 +128,54 @@ private void SetAbilities(AceCreatureObject aco)
             abilities.Add(Enum.Ability.Mana, Mana);
 
             Abilities = new ReadOnlyDictionary<Enum.Ability, CreatureAbility>(abilities);
+
+            IsAlive = true;
+        }
+
+        public void Kill(Session session)
+        {
+            IsAlive = false;
+
+            // Send UpdateHealth Message with 0 Health - this is not necessary according to live pcaps
+            // var healthLossEvent = new GameEventUpdateHealth(session, Guid.Full, 0f);
+            // session.Network.EnqueueSend(healthLossEvent);
+
+            // 1. step in retail: Create and send the death notice
+            string killMessage = $"{session.Player.Name} has killed {Name}.";
+            var creatureDeathEvent = new GameEventDeathNotice(session, killMessage);
+            session.Network.EnqueueSend(creatureDeathEvent);
+
+            // 2. step in retail: SoundEvent: sound 48 (HitFlesh1), volume 0,5
+            session.Player.ActionApplySoundEffect(Sound.HitFlesh1, Guid);
+
+            // 3. step in retail: SoundEvent: sound 14 (Wound3), volume 1
+            session.Player.ActionApplySoundEffect(Sound.Wound3, Guid);
+
+            // 4. step in retail: MovementEvent: (Hand-)Combat to Death
+            // TODO: this isn't really working yet, we need more work on the motion stuff
+            GeneralMotion motion1 = new GeneralMotion(MotionStance.Standing, new MotionItem(MotionCommand.Dead));
+            session.Network.EnqueueSend(new GameMessageUpdateMotion(this, session, motion1));
+
+            // 5. step in retail: Play Script to show wound depending on where the last hit was
+            // TODO: this is not showing yet, guess we need to give all the sounds and scripts some time to actually play
+            var effectEvent = new GameMessageScript(Guid, Network.Enum.PlayScript.SplatterMidRightFront);
+            session.Network.EnqueueSend(effectEvent);
+
+            // 6. step in retail: UpdatePosition: gotta verify this on more examples
+            // I guess it lowers the z axis, so the corpse isn't hovering in about the midth of a usual drudge
+
+            // 7. step in retail: DeleteObject Creature
+            // Remove Creature from Landblock this sends GameMessageRemoveObject
+            LandblockManager.RemoveObject(this);
+
+            // 8. step in retail: Create Corpse (dead for now - but guess is standing or in combat should be right)
+            // TODO: set text of killer in description
+            var corpse = CorpseObjectFactory.CreateCorpse(this, this.Location);
+            LandblockManager.AddObject(corpse);
```

Comment:
As a proof of concept, changing the code above to this gets you where you wanted to go
```c            
            Thread.Sleep(1000);

            var corpse = CorpseObjectFactory.CreateCorpse(this, this.Location);
            corpse.Location.PositionX += .6f;
            corpse.Location.PositionY += corpse.PhysicsData.ObjScale + .2f - .4f;

            LandblockManager.RemoveObject(this);
            LandblockManager.AddObject(corpse);
```